modules/sb-start-tsb.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * applications/service_brokers/installing-template-service-broker.adoc
+
+[id='sb-start-tsb-{context}']
+= Starting the template service broker
+
+Once you have installed the template service broker Operator, you can start the
+template service broker using the following procedure.
+
+.Prerequisites
+
+* You have enabled service catalog.
+* You have installed the template service broker Operator.
+
+.Procedure
+
+. Navigate in the web console to *Catalog* -> *Installed Operators* and select the `template-service-broker` project.
+. Select the *Template Service Broker Operator*.
+. Under *Provided APIs*, click *Create New* for *Template Service Broker*.
+. Review the default YAML and click *Create*.
+
+Once the template service broker has started, you can view the available
+template applications by navigating to *Catalog* -> *Developer Catalog* and
+selecting the *Service Class* checkbox. Note that it may take a few minutes for
+the template service broker to start and the template applications to be available.
+
+If you do not yet see these Service classes, you can check the status of the
+following items:
+
+* Template service broker Pod status
+** From the *Workloads* -> *Pods* page for the *template-service-broker*
+project, verify that the Pod that starts with `apiserver-` has a status of
+*Running* and readiness of *Ready*.
```

Comment:
Add checking of clusterservicebroker status. 
from UI:
Catalog-> Broker Management-> Service Brokers
status of 'template-service-broker ' should be ready.